lighthouse-cli/test/smokehouse/oopif-expectations.js

```diff
@@ -16,10 +16,11 @@ module.exports = [
       'network-requests': {
         details: {
           items: {
-            // The page itself only makes a few requests.
-            // We want to make sure we are finding the iframe's requests (airhorner) while being flexible enough
-            // to allow changes to the live site.
-            length: '>10',
+            // We want to make sure we are finding the iframe's requests (paulirish.com) *AND*
+            // the iframe's iframe's iframe's requests (youtube.com/doubleclick/etc).
+            // - paulirish.com ~50-60 requests
+            // - paulirish.com + all descendant iframes ~80-90 requests
+            length: '>70',
```

Comment:
I have a feeling this is going to have a big effect on TTI as network idle is likely going to be pushed out in a bunch of cases.